packages/core/src/lib/jdk/KeyTool.ts

```diff
@@ -100,6 +100,11 @@ export class KeyTool {
     }
     const keyListCmd = [
       'keytool',
+      // Forces the language to 'en' in order to get the expected formatting.
+      // The JVM seems to ignore the LANG and LC_ALL variables, so we set the value
```

Comment:
I'm assuming the JVM wants something that is consistent across OSes? (Do `LANG` and `LC_ALL` exist on Windows?)

Reply:
It seems `LANG` does exist on Windows. Nevertheless, this seems to the the appropriate way to do this.